_pages/research.md

```diff
@@ -24,4 +24,44 @@ mechanics (QM), molecular dynamics (MD), and rare-event sampling methods to unco
 fundamental understandings of protein-carbohydrate structure-function relationships, 
 opening opportunities for rational design of enzymes and diagnostic tools. Collaborating 
 with experimental groups, we aim to understand past and guide future wet-lab studies to 
-advance renewable chemicals and fuels as well as health.
\ No newline at end of file
+advance renewable chemicals and fuels as well as health.
+
+Currently, these interests fuel the following projects:
+
+# Computational Tools for Protein Design, Led by [Alex Adams](../team#xadams) 
+
+<img align="left" src="../assets/img/xyle.jpg" style="margin:10px 10px 0px 0px">
+A challenge for utilizing biomass for renewable fuels and chemicals is achieving co-utilization of the 
+primary constituent sugars: glucose and xylose. Typically, microorganisms selectively catabolize all 
+available glucose before consuming xylose, leading to two distinct growth phases which complicate design 
+of an economical continuous process. We aim to investigate the molecular basis of glucose inhibition of 
+xylose transport into cells through multi-scale atomistic modeling. We will apply this understanding to 
+rationally design improved protein transporters that advance biotechnology and future protein design.
+
+# Size-based sorting of Exosomes, Led by [Emma Purcell](../team#epurcell)
+
+Exosomes are lipid bilayer membrane vesicles that are known to enhance tumor metastasis by stimulating both 
+tumor growth and tumor cell migration. It is crucial to be able to sort exosomes out of blood to further 
+elucidate their role in cancer. Current techniques for exosomal sorting in microfluidics are expensive and 
+low throughput, using either antibody capture or other active techniques.  By designing a high throughput, 
```

Comment:
Make "low throughput either "have low throughput" or "low-throughput"  

Also, the end of the sentence should more explicitly link the methods to the description in the first part.  Perhaps change "using" to "due to their using".

We should probably also make "high throughput" "high-throughput", thus suggesting the use of "low-throughput" for the first suggetion.